lib/facilities/multi_client.rb

```diff
@@ -15,7 +16,7 @@ def run(requests)
       requests.map do |req|
         if req.response.timed_out?
           Rails.logger.error "GIS request timed out: #{req.url}"
-          raise Common::Client::Errors::RequestTimeout
+          raise Common::Exceptions::ClientError.new('Facility request timed out', {})
```

Comment:
Hmmm, this is interesting. I think since you have multiple requests, you should maybe loop through, collect all the errors, and then maybe build it using an error exception that includes an array of all of the errors. Right now, this would just return JSON for a single error and break out of the loop.